microsoft-edge/devtools-guide-chromium/whats-new/2023/03/devtools-111.md

```diff
@@ -13,17 +13,17 @@ ms.date: 03/15/2023
 
 
 <!-- ====================================================================== -->
-## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens devices
+## Remote debugging of Microsoft Edge, PWAs, and WebView2 controls on Xbox and HoloLens 2 devices
 
 <!-- Subtitle: Get the Remote Tools for Microsoft Edge app from the Microsoft Store to set up remote debugging today! -->
 
-The **Remote Tools for Microsoft Edge** app is now available on Xbox and HoloLens devices.  The **Remote Tools for Microsoft Edge** app is a plug-in for the **Windows Device Portal for Desktop** that enables remote debugging for tabs in Microsoft Edge, Progressive Web Apps (PWAs), and WebView2 controls.  By using remote debugging, you can run web content on a remote device and debug it right from your development machine.
+The **Remote Tools for Microsoft Edge** app is now available on Xbox and HoloLens 2 devices.  The **Remote Tools for Microsoft Edge** app is a plug-in for the **Windows Device Portal for Desktop** that enables remote debugging for tabs in Microsoft Edge, Progressive Web Apps (PWAs), and WebView2 controls.  By using remote debugging, you can run web content on a remote device and debug it right from your development machine.
 
 ![The Remote Tools for Microsoft Edge app in the Microsoft Store](./devtools-111-images/remote-tools-in-the-microsoft-store.png)
 
-With remote debugging enabled, you don't need to open DevTools on either HoloLens or Xbox devices any more.  Instead, you can connect to the remote Xbox or HoloLens device using the Device portal, open the `edge://inspect` page, and then select the target (tab, PWA, or WebView2 control) that you want to use with DevTools.
+With remote debugging enabled, you don't need to open DevTools on either HoloLens 2 or Xbox devices any more.  Instead, you can connect to the remote Xbox or HoloLens 2 device using the Device portal, open the `edge://inspect` page, and then select the target (tab, PWA, or WebView2 control) that you want to use with DevTools.
 
-To give us feedback on the remote debugging flow for Microsoft Edge, leave a comment in [Issue 136: Remote debugging Xbox and HoloLens devices is now available with the latest version of the Remote Tools for Microsoft Edge app](https://github.com/MicrosoftEdge/DevTools/issues/136).
+To give us feedback on the remote debugging flow for Microsoft Edge, leave a comment in [Issue 136: Remote debugging Xbox and HoloLens 2 devices is now available with the latest version of the Remote Tools for Microsoft Edge app](https://github.com/MicrosoftEdge/DevTools/issues/136).
```

Comment:
@zoherghadyali @nishitha-burman -- Should Zoher change the title of the Issue to add "2"?  Right now this link text here doesn't match the destination.
The link text in this PR:
"Issue 136: Remote debugging Xbox and HoloLens 2 devices is now available with the latest version of the Remote Tools for Microsoft Edge app"
The actual destination title/heading at https://github.com/MicrosoftEdge/DevTools/issues/136 --
"Remote debugging Xbox and HoloLens devices is now available with the latest version of the Remote Tools for Microsoft Edge app"

Reply:
We should specify HoloLens 2

Reply:
ok then this PR should specify "HoloLens 2" in this link text regardless of whether/when Zoher adds "2" to the Issue's title.

Reply:
Title updated.

Reply:
title of Zoher's issue thread lacks '2'
https://github.com/MicrosoftEdge/DevTools/issues/136
"Remote debugging Xbox and HoloLens devices is now available with the latest version of the Remote Tools for Microsoft Edge app"

you probably mean link text has been updated (maybe locally and will push soon)

Reply:
No, sorry. I thought you were referring to my PR.

Reply:
The latest Files Changed tab shows
To give us feedback on the remote debugging flow for Microsoft Edge, leave a comment in [Issue 136: Remote debugging Xbox and HoloLens 2 devices is now available with the latest version of the Remote Tools for Microsoft Edge app](https://github.com/MicrosoftEdge/DevTools/issues/136).

therefore you can close this thread